docs/state/var-operations.md

```diff
@@ -0,0 +1,529 @@
+                    ],
+                variant="striped",
+                color_scheme="red",
+            ),
+        ),
+        rx.button("Update", on_click=VarDivState.update),
+    )
+```
+
+```python eval
+docdemo_from(VarDivState, component=var_div_example, imports=["import random"])
+```
+
+
+### And, Or and Not
+
+In Reflex the `&` operator represents the logical AND when used in the front end. This means that it returns true only when both conditions are true simultaneously. 
+The `|` operator represents the logical OR when used in the front end. This means that it returns true when either one or both conditions are true.
+The `~` operator is used to invert a var. It is used on a var of type `bool` and is equivalent to the `not` operator.
+
+```python exec
+import random
+
+class VarLogicalState(State):
+    var_1: bool = True
+    var_2: bool = True
+
+    def update(self):
+        self.var_1 = random.choice([True, False])
+        self.var_2 = random.choice([True, False])
+
+def var_logical_example():
+    return rx.vstack(
+        rx.table_container(
+            rx.table(
+                headers=["Var 1", "Var 2", "Operation", "Outcome"],
+                rows=[
+                    (f"{VarLogicalState.var_1}", f"{VarLogicalState.var_2}", "Logical AND (&)", f"{VarLogicalState.var_1 & VarLogicalState.var_2}"),
+                    (f"{VarLogicalState.var_1}", f"{VarLogicalState.var_2}", "Logical OR (|)", f"{VarLogicalState.var_1 | VarLogicalState.var_2}"),
+                    (f"{VarLogicalState.var_1}", f"{VarLogicalState.var_2}", "The invert of Var 1 (~)", f"{~VarLogicalState.var_1}"),
+                    ],
+                variant="striped",
+                color_scheme="green",
+            ),
+        ),
+        rx.button("Update", on_click=VarLogicalState.update),
+    )
+```
+
+```python eval
+docdemo_from(VarLogicalState, component=var_logical_example, imports=["import random"])
+```
+
+### Contains, Reverse and Join
+
+
+The 'in' operator is not supported for Var types, we must use the `Var.contains()` instead. When we use `contains`, the var must be of type: `dict`, `list`, `tuple` or `str`. 
+`contains` checks if a var contains the object that we pass to it as an argument.
+
+We use the `reverse` operation to reverse a list var. The var must be of type `list`.
+
+Finally we use the `join` operation to join a list var into a string. 
+
+```python exec
+class VarListState(State):
+    list_1: list = [1, 2, 3, 4, 6]
+    list_2: list = [7, 8, 9, 10]
+    list_3: list = ["p","y","t","h","o","n"]
+
+def var_list_example():
+    return rx.vstack(
+        rx.heading(f"List 1: {VarListState.list_1}", size="lg"),
+        rx.heading(f"List 2: {VarListState.list_2}", size="lg"),
+        rx.heading(f"List 3: {VarListState.list_3}", size="lg"),
+        
+        rx.heading(f"Does List 1 contain the number 3: {VarListState.list_1.contains(3)}", size="md"),
```

Comment:
I think a clean way to display this section would be to have an 

hstack(
vstack(
rx.heading(f"List 1: {VarListState.list_1}", size="lg"),
rx.heading(f"Contains 3: {VarListState.list_1.contains(3)}", size="md"),
),
vstack(
rx.heading(f"List 2: {VarListState.list_2}", size="lg"),
rx.heading(f"Reverse List 2: ...),
)
...
)
 
 To better show the correlated parts. 
 
 Can shorten these comments
 Does List 1 contain the number 3: true  -> List 1 Contains 3: True
Join List 3 into string: python -> List 3 Joins: python

Reply:
done